microservices/frame_engine/install_requirements.sh

```diff
@@ -0,0 +1,8 @@
+apt-get update
+apt-get install build-essential -y
+apt-get install apt-utils  -y
+
+python -m pip install opencv-python
+python -m pip install camera
+apt-get install python-picamera python3-picamera
+python -m pip install requests
```

Comment:
Similarly to what I said in `underwater_od` I think is safer if you specify the versions you actually want to install. Sometimes there's an update breaking functionality and it could happen when we least want it in scripts like this.

Reply:
solved it for the API microservice